otherlibs/runtime_events/runtime_events.mli

```diff
@@ -133,6 +133,60 @@ module Timestamp : sig
     val to_int64 : t -> int64
 end
 
+module Type : sig
+  type 'a t
+  (** The type for an user event content type *)
+
+  val event : unit t
+  (** An event has no data associated with it *)
+
+  type span = Begin | End
+
+  val span : span t
+  (** An event that have a beginning and an end *)
+
+  val counter : int t
+  (** A counter is a number *)
```

Comment:
nit: If a counter is an integer, can we just call it an `int`? The naming seems to suggest that all integers are counters. It may just be that we're following what other frameworks call such an event: https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/preview#heading=h.msg3086636uq.